### PR TITLE
Added Sysdig monitor

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [X] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by nmohantyctr — APPROVED** [review]

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+id: sysdig-monitor-source
+title: Sysdig Monitor Source
+sidebar_label: Sysdig Monitor
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Sysdig Monitor Cloud-to-Cloud source provides the ability to collect events such as monitoring, alerting, troubleshooting from Sysdig Monitor platform and send them to Sumo logic. It ingests events from the [Sysdig Monitor API](https://app.sysdigcloud.com/api/public/docs/index.html).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md:9]:
```suggestion
The Sysdig Monitor Cloud-to-Cloud source provides the ability to collect events such as monitoring, alerting, troubleshooting from the Sysdig Monitor platform and send them to Sumo Logic. It ingests events from the [Sysdig Monitor API](https://app.sysdigcloud.com/api/public/docs/index.html).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+## Prerequisites
+
+Before you begin setting up your **Sysdig Monitor** Source, the integration must be configured with the **Base URL** and **Sysdig Monitor API Token**.
+1. Base URL.
+2. API Token.
+
+### Base URL
+
+The **Base URL** is the URL where your Sysdig-Monitor account is located  To get the base URL, follow the steps below:
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md:19]:
```suggestion
The **Base URL** is the URL where your Sysdig Monitor account is located. To get the base URL, follow the steps below:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+   | US2 |	`https://us2.app.sysdig.com/` |
+   | EU1 |	`https://eu1.app.sysdig.com/` |
+   | AU1 |	`https://app.au1.sysdig.com/` |
+   | US4 |	`https://app.us4.sysdig.com/` |
+
+
+### API Token
+
+The **API security token** is used to authenticate with Sysdig HTTP API. To get the **Sysdig API token**, follow the steps below:
+1. Log into Sysdig Monitor application.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md:36]:
```suggestion
1. Log in to the Sysdig Monitor application.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+The Sysdig Monitor platform provides security, monitoring, and forensics in a cloud, container, and microservices-friendly architecture, integrated with Kubernetes and Docker. It collects, correlates, and visualizes full-stack data and provides monitoring dashboards.
+
+When a Sysdig Monitor Source is created, it goes through the following states:
+
+1. **Pending**. Once the Source is submitted, it is validated, stored, and placed in a **Pending** state.
+1. **Started**. A collection task is created on the Hosted Collector.
+1. **Initialized**. The task configuration is complete in Sumo Logic.
+1. **Authenticated**. The Source successfully authenticated with Sysdig Monitor.
+1. **Collecting**. The Source is actively collecting data from SysDIG Monitor.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md:62]:
```suggestion
1. **Collecting**. The Source is actively collecting data from Sysdig Monitor.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+If the Source has any issues during any one of these states, it is placed in an **Error** state. When you delete the Source, it is placed in a **Stopping** state. When it has successfully stopped, it is deleted from your Hosted Collector.
+On the [Collection page](/docs/manage/health-events#collection-page), the Health and Status for Sources is displayed. Use [Health Events](/docs/manage/health-events.md) to investigate issues with collection.<br/>![Azure Event Hubs error.png](/img/send-data/Azure-Event-Hubs-error.png)
+
+Hover your mouse over the status icon to view a tooltip with a count of the detected errors and warnings. You can click on the status icon to open a Health Events panel with details on each detected issue.<br/>![health error generic.png](/img/send-data/azure_health_error_generic.png)
+
+
+## Setup and configuration
+
+When you create a Sysdig-Monitor Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md:72]:
```suggestion
When you create a Sysdig Monitor Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+| ThirdPartyGeneric | Normally due to an error communicating with the third party service APIs.                                     | Yes                                                   | The Source will retry for up to 90 minutes, after which it quits.                               | ThirdPartyGenericError |
+| FirstPartyGeneric | Normally due to an error communicating with the internal Sumo Logic APIs.                                     | Yes                                                   | The Source will retry for up to 90 minutes, after which it quits.                               | FirstPartyGenericError |
+
+### JSON configuration
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details. 
+
+| Parameter | Type | Required | Description | Access |
+|--|--|--|--|--|
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md:101]:
```suggestion
|:--|:--|:--|:--|:--|
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+| `config` | JSON Object  | Yes | Contains the [configuration-parameters](#config-parameters) of the Source. |
+| `schemaRef` | JSON Object  | Yes | Use `{"type":"Sysdig Monitor Source"}` for Sysdig Monitor Source. | not modifiable |
+| `sourceType` | String | Yes | Use `Universal` for Sysdig Monitor Source. | not modifiable |
+
+### Config Parameters
+
+| Parameter | Type | Required | Description | Access |
+|--|--|--|--|--|
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md:109]:
```suggestion
|:--|:--|:--|:--|:--|
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+| `name` | String | Yes | Type the desired name of the Source and it must be unique per Collector. This value is assigned to the `metadata field _source`.  | modifiable |
+| `description` | String  | No | Type the description of the Source. | modifiable |
+| `category` | String | No | Type the category of the source. This value is assigned to the metadata field `_sourceCategory`. | modifiable |
+| `fields` | JSON Object | No | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field `_siemForward` to enable forwarding to SIEM. | modifiable |
+| baseURL | String | Yes | Region URL of the Sysdig monitor application. | modifiable |
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md:114]:
```suggestion
| `baseURL` | String | Yes | Region URL of the Sysdig monitor application. | modifiable |
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+| apiToken | String | Yes | Secret api key to authenticate your account.  | modifiable |
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md:115]:
```suggestion
| `apiToken` | String | Yes | Secret api key to authenticate your account.  | modifiable |
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Style/formatting changes requested

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md (hunk 7 above)
Comment (JainM6) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md:108]: These colons are just to fix the alignment. In this case, these columns would work without these colons as well. So no harm even if we do not apply these colons here.

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
@JainM6 , is this ready to merge?

**Comment by JainM6** [comment]
We are waiting to get more information from the customer side.

**Comment by JainM6** [comment]
This is not going to release as this C2C has been canceled.

**Pull request closed without merging**